src/settings.js

```diff
@@ -311,8 +312,11 @@ var STB_IMAGE = 0; // Enables building of stb-image, a tiny public-domain librar
                    // When enabled, stb-image will be used automatically from IMG_Load and IMG_Load_RW. You
                    // can also call the stbi_* functions directly yourself.
 
-var LEGACY_VM_SUPPORT = 0; // Enable this to get support for non-modern browsers, node.js, etc. This adds:
-                           //  * Polyfilling for Math.clz32, Math.trunc, Math.imul, Math.fround
+var LEGACY_VM_SUPPORT = 0; // Enable this to get support for non-modern browsers, node.js, etc. This gives you
+                           // the highest possible probability of the code working everywhere, even in rare old
+                           // browsers and shell environments. Specifically:
+                           //  * Add polyfilling for Math.clz32, Math.trunc, Math.imul, Math.fround.
+                           //  * Disable WebAssembly.
```

Comment:
Disabling WebAssembly when `-s LEGACY_VM_SUPPORT=1` makes me feel sad about the existence of the whole `LEGACY_VM_SUPPORT` setting overall, it might be on the road to becoming a bit of a "all non-latest features pile in here" type of setting.

Does `-s LEGACY_VM_SUPPORT=1 -s WASM=1` work to give one backwards compatibility while also targeting Wasm?

Reply:
Good point, in the long term maybe we want to split it out. For now, though, afaik it is used by people to target things like IE11, and other really older browsers without Math.clz etc. - those definitely also don't have wasm. So the setting is meaningless with wasm enabled, currently.